src/components/DetailedRoute/DetailedRoute.tsx

```diff
@@ -0,0 +1,66 @@
+// import React, {FC, Dispatch, SetStateAction} from 'react';
+import { Link } from 'react-router-dom';
+import { detailedRouteData } from '../../mockData';
+import userPhoto from '../../assets/images/man.png';
+import exampleMapOne from '../../assets/images/exampleMapOne.png'
+import exampleMapTwo from '../../assets/images/exampleMapTwo.png'
+
+// interface DetailedRouteProps {
+//   id: string
+// }
+
+// const DetailedRoute: FC<DetailedRouteProps> = () => {
+const DetailedRoute = () => {
+
+  const days = detailedRouteData.route.days.map(day => {
+    return (
+      <div className='day'>
+        {(day === 'tuesday' || day === 'thursday') ? day.charAt(0).toUpperCase() + day.slice(1,2) : day.substring(0, 1).toUpperCase()}
```

Comment:
Maybe we can add a conditional for Saturday and Sunday here

Reply:
agreed